### ISSUE TITLE
Resolving a conflicted post offline doesn't auto-upload the post when online

### ISSUE THREAD

**Issue by oguzkocer** [wordpress-mobile/WordPress-Android#10527]
If a post with unhandled auto save state is changed by the user and saved, it never uploads to the server. I am able to reproduce this for a draft, but I suspect it'd also happen for other statuses. Here are the steps:

1. Create a post with `Unhandled auto save` state - I think all you need to do is to just update a post from both the Android client and another one. I used a post that resulted from the bug #10526
2. Turn on airplane mode
3. Tap on the post from `Step 1` and select "The version from another device" (this shouldn't matter)
4. Use the overflow menu to save the post (you can make changes to it, I don't think it'd matter)
5. Notice that the post says "Draft will be saved to the server next time your device is online"
6. Turn off airplane mode
7. Profit (nothing happens)

Here is my test: https://cloudup.com/csnK5FUNo-V

_This test was run with a WordPress.com account._

_**P.S:** I think the choice of red color for the "Draft will be saved to the server next time your device is online" makes it feel like it's an error. Maybe we can use a different color for this?_

**Comment by yaelirub** [comment]
>P.S: I think the choice of red color for the "Draft will be saved to the server next time your device is online" makes it feel like it's an error. Maybe we can use a different color for this?

I agree. Can we change it to the mustard yellow color we use for "post will be published...? 
https://user-images.githubusercontent.com/50576199/60972121-8ad36680-a31d-11e9-930b-e5d3f3fd79fc.jpg

**Comment by jkmassel** [comment]
Howdy folks! I'm freezing `13.4` today, so this is being bumped to `13.5`. If you'd like it to go out as part of `13.4`, please target `release/13.4` for this PR, and DM me once it's merged – I'll be happy to cut a new beta release!

**Comment by jkmassel** [comment]
Howdy folks! I'm freezing `13.5` today, so this is being bumped to `13.6`. If you'd like it to go out as part of `13.5`, please target `release/13.5` for this PR, and DM me once it's merged – I'll be happy to cut a new beta release!

**Comment by jkmassel** [comment]
Howdy folks! I'm freezing `13.6` today, so this is being bumped to `13.7`. If you'd like it to go out as part of `13.6`, please target `release/13.6` for this PR, and DM me once it's merged – I'll be happy to cut a new beta release!

**Comment by maxme** [comment]
Sounds like this one is bumped at every release cycle, I'll remove the milestone for now - cc @yaelirub @oguzkocer

**Comment by jd-alexander** [comment]
I tested this on WPAndroid alpha-285, Pixel 3a, Android 11.

I used the steps here in an attempt to reproduce this issue. I am happy to report that it no longer exists as there's now a network connection check that is done before allowing post edits while the app is within this state, so I am going to be closing this issue.

<kbd><img src="https://user-images.githubusercontent.com/1509205/113089381-2d481300-91ad-11eb-9f60-a12732b95a2a.gif" width="320"></kbd>
